data/json/itemgroups/stashes.json

```diff
@@ -68,5 +68,218 @@
       { "item": "diazepam", "prob": 1, "charges": [ 1, 10 ] },
       { "item": "lsd", "prob": 6, "charges": [ 1, 5 ] }
     ]
+  },
+  {
+    "type": "item_group",
+    "id": "bugout_bag",
+    "subtype": "distribution",
+    "//": "randomized assortment of possible bugout bags people would have as the apocalypse ramped up"
+    "entries": [
+      { "group": "small_bugout", "prob": 50 },
+      { "group": "medium_bugout", "prob": 40 },
+      { "group": "heavy_bugout", "prob": 10 }
+    ]
+  },
+  {
+    "id": "small_bugout",
+    "type": "item_group",
+    "//": "a light bugout, stuff people would have on hand, not super full",
+    "container-item": "backpack",
```

Comment:
Is it possible to make this a variety of packs(ex. also possible to spawn inside a daypack or leather backpack)?

Reply:
hmmm it'll be few more lines of code but I suppose I could 😄